plugins/osdn/node.go

```diff
@@ -139,6 +148,46 @@ func (node *OsdnNode) Start() error {
 	return nil
 }
 
+const cniConfigPath = "/etc/cni/net.d/80-openshift-sdn.conf"
+
+func writeCniConfig() error {
+	const config string =`{
+  "cniVersion": "0.1.0",
+  "name": "openshift-sdn",
+  "type": "openshift-sdn"
+}`
+
+	return ioutil.WriteFile(cniConfigPath, []byte(config), 0600)
```

Comment:
We should not be writing files to /etc at runtime. That should be installed with the plugin.